showcase/src/assets/examples/reactive-forms-errors/reactive-forms-errors.html

```diff
@@ -0,0 +1,141 @@
+<form [formGroup]="formGroup" (ngSubmit)="onSubmitUserDetails(formGroup)" class="form-group">
+	<div class="grid">
+		<app-card class="form-card">
+			<mat-card-content>
+				<mat-form-field class="full-width">
+					<input matInput [placeholder]="'SHOWCASE.FORMS.NGX_FORMS.USER_NAME' | translate" formControlName="username" required />
+					<mat-error>
+						<ng-template ngxFormErrors="username" #usernameField="ngxFormErrors"></ng-template>
+					</mat-error>
+				</mat-form-field>
+
+				<div formGroupName="matchingPasswords" ngxFormErrorsGroup="passwords-group">
+					<mat-form-field class="full-width">
+						<input
+							matInput
+							type="password"
+							[placeholder]="'SHOWCASE.FORMS.NGX_FORMS.PASSWORD' | translate"
```

Comment:
Can you replace this translation key by the translated text?

Reply:
Sorry, I think I don't get it :-(
You mean, I shouldn't use the translation here ?